dotCMS/src/main/java/com/dotcms/ai/app/AppConfig.java

```diff
@@ -260,13 +263,24 @@ public AIModel resolveModel(final AIModelType type) {
      * @param modelName the name of the model to find
      */
     public AIModel resolveModelOrThrow(final String modelName) {
-        return AIModels.get()
+        final AIModel model = AIModels.get()
                 .findModel(host, modelName)
                 .orElseThrow(() -> {
                     final String supported = String.join(", ", AIModels.get().getOrPullSupportedModels());
                     return new DotRuntimeException(
                             "Unable to find model: [" + modelName + "]. Only [" + supported + "] are supported ");
                 });
+
+        if (!model.isOperational()) {
+            Logger.debug(
```

Comment:
may be error

Reply:
@victoralfaro-dotcms you can also call the `debugLogger(final Class<?> clazz, final Supplier<String> message)` method in here if necessary, which uses the `com.dotcms.ai.debug.logging` config property to log information.